openpathsampling/pathmover.py

```diff
@@ -320,11 +333,8 @@ def move(self, globalstate):
 
         return paths.EmptyPathMoveChange()  # pragma: no cover
 
-    def __str__(self):
-        if self.name == self.__class__.__name__:
-            return self.__repr__()
-        else:
-            return self.name
+    def __repr__(self):
+        return self.name + '(%s)' % hex(id(self))[2:]
```

Comment:
At the very least, at a "0x" before the hex string. That makes it clear that this is a hex representation. Also, I would suggest adding a space before the parentheses.

This whole thing will need to be revisited when we get around to #233.